setup.py

```diff
@@ -0,0 +1,41 @@
+import os
+from setuptools import setup, find_packages
+
+
+here = os.path.abspath(os.path.dirname(__file__))
+
+
+def content(path, cwd=None):
```

Comment:
Why is the `cwd` argument sometimes necessary?

Reply:
I was having issues in reading the README file vs the requirements one, it was not able to read the readme file withtout the current working dir var.

Reply:
OK but why not always use `here` when building the paths, then? This way youd have one less parameter and you'll know all paths are in the same directory as this file (I like removing things ;-) )